src/python/tests/data_purge/test_data_purge.py

```diff
@@ -0,0 +1,391 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import asyncio
+from collections import OrderedDict
+from datetime import datetime, timezone, timedelta
+import pytest
+import random
+import uuid
+import json
+
+from foglamp import configuration_manager
+from foglamp.data_purge.purge import start, _CONFIG_CATEGORY_NAME
+from foglamp.storage.payload_builder import PayloadBuilder
+from foglamp.storage.storage import Storage, Readings
+
+__author__ = "Ori Shadmon"
+__copyright__ = "Copyright (c) 2017 OSI Soft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_core_management_port = 36979
```

Comment:
This will not always work as the port will change. A better approach for tests would be to tun the storage service with a fixed port, which can be achieved via configuration options, and use that.

Please add a TODO that this needs to be addressed.